docs/section23/Rule23-11.md

```diff
@@ -0,0 +1,47 @@
+
+# Airside System - Rule 23-11 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** True  
+**Rule ID:** 23-11  
+**Rule Description:** System 11 Supply air temperature shall be reset from minimum supply air temp at 50% cooling load to room temp at 0% cooling load.  OR the SAT is reset higher by 5F under minimum cooling load conditions.    
```

Comment:
rule has been updated to included the System 11 logic from 23-2.

This is being evaluated as mandatory based on an email from Jason Glazer stating that we can suggest clarifications for FanSystemTemperatureControlOptions based on how these parameters are used in the RCT.  I created a new issue suggesting clarifying language:

https://github.com/open229/ruleset-model-description-schema/issues/219